share/revng/pipelines/revng-pipelines.yml

```diff
@@ -148,14 +152,10 @@ Branches:
             Therefore, they lack arguments and return values.
       - Name: enforce-abi
         Pipes:
-          - Type: llvm-pipe
-            UsedContainers: [module.bc.zstd]
-            Passes:
-              - drop-root
```

Comment:
I was like "nononono", but it's "yesyesyes".

We should have done this much sooner.